src/python/excel_uploader_main.py

```diff
@@ -90,12 +90,13 @@ def is_uploading_local_files(self, files):
         if len(files) < 1:
             return False
 
-        for key in files:
-            # if file starts with http or https, then file is local, and returns true
-            if not files[key].source.startswith("http://") and not files[
-                key
-            ].source.startswith("https://"):
-                return True
+        for key, values in files.items():
+            for ix in range(len(values)):
+                # if file starts with http or https, then file is local, and returns true
```

Comment:
I think this may have been a small type

```python
# if file does NOT start with http or https then it is a local file
```

we could even write `if file does not start with "http"` that way it works for both http and https